src/clone/clone-utils.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace EE\Site\Cloner\Utils;
+
+use EE;
+use EE\Site\Cloner\Site;
+use function EE\Utils\get_flag_value;
+use function EE\Utils\random_password;
+use function EE\Utils\trailingslashit;
+
+function copy_site_db( Site $source, Site $destination ) {
+
+	$site_type = $source->site_details['site_type'];
+	$db_host   = $source->site_details['db_host'];
+
+	if ( 'wp' === $site_type || 'php' === $site_type && ! empty( $db_host ) ) {
+		$source_site_name      = $source->site_details['site_url'];
+		$destination_site_name = $destination->site_details['site_url'];
+
+		EE::log( 'Exporting database from source' );
+
+		$filename       = $source_site_name . '-' . random_password() . '.sql';
+		$export_command = 'ee shell --skip-tty ' . $source_site_name . ' --command=\'wp db export ../' . $filename . '\'';
+
+		if ( $source->execute( $export_command )->return_code ) {
+			EE::error( 'Unable to export database on source. Please check for file system permissions and disk space.' );
+		}
+
+		EE::log( 'Copying database to destination' );
+
+		$source_fs_path      = trailingslashit( $source->site_details['site_fs_path'] ) . 'app/' . $filename;
+		$destination_fs_path = trailingslashit( $destination->site_details['site_fs_path'] ) . 'app/' . $filename;
+
+		$copy_db_command = rsync_command( $source->get_rsync_path( $source_fs_path ), $destination->get_rsync_path( $destination_fs_path ) );
+
+		if ( ! EE::exec( $copy_db_command ) ) {
+			EE::error( 'Unable to copy database to destination. Please check for file system permissions and disk space.' );
+		}
+
+		EE::log( 'Importing database in destination' );
+
+		$import_command = 'ee shell --skip-tty ' . $destination_site_name . ' --command=\'wp db import ../' . $filename . '\'';
```

Comment:
Again, will fail for `php` site type.

Reply:
Fixed in https://github.com/EasyEngine/site-command/pull/357/commits/8260b82ad98c8562bc5c0513609a4906c84df2c5